src/Protocol/PortableContact.php

```diff
@@ -1375,7 +1375,12 @@ public static function checkServer($server_url, $network = "", $force = false)
 					}
 
 					$info = $data['info'];
-					$register_policy = constant($data['register_policy']);
+					if (in_array($data['register_policy'], ['REGISTER_CLOSED', 'REGISTER_APPROVE', 'REGISTER_OPEN'])) {
+						$register_policy = constant($data['register_policy']);
+					} else {
+						Logger::log("Register policy '".$data['register_policy']."' from $server_url is invalid.");
```

Comment:
If you're going for double-quotes, you can write `"Register policy '{$data['register_policy']}' from $server_url is invalid."`.

Either that or wrap operators with spaces.